debian/control

```diff
@@ -116,8 +116,7 @@ Description: common ceph daemon libraries and management tools
 Package: ceph-mds
 Architecture: linux-any
 Depends: ceph-base (= ${binary:Version}), ${misc:Depends}, ${shlibs:Depends}
-Recommends: ceph-fs-common (= ${binary:Version}),
-            ceph-fuse (= ${binary:Version}),
+Recommends: ceph-fuse (= ${binary:Version}),
```

Comment:
this is the 11st scenario in Renaming_a_Package[1]. so probably, we should mark ceph-common 

> Breaks: ceph-fs-common (<< 11.0) (for backport ease)
> Replaces: ceph-fs-common (<< 11.0)

i don't think we need the "Provides: ceph-fs-common"  flag, as the only package depends on ceph-fs-common is ceph-fs-common-dbg. and apparently, they version should be identical. so this does not apply in this case.

---
[1] https://wiki.debian.org/PackageTransition